model/src/main/java/com/redhat/ipaas/model/connection/Connector.java

```diff
@@ -45,6 +45,9 @@ default Kind getKind() {
 
     String getDescription();
 
+    Optional<String> getCamelConnectorGAV();
```

Comment:
A _CamelConnector_ correspond to an `Action`. So would call it `getVerifierAction()` and return the corresponding `Action` from where the GAV is optained. See #219 for a discussion about that.